### ISSUE TITLE
Almost there!

### ISSUE THREAD

**Issue by zhmrab** [forknote/forknote#60]
Hey everyone! I set up my seed nodes using 2 VPS and after starting the daemon on my computer I get this : 

SwappedVector cache hits: 1, misses: 1 (50.00%)
SwappedVector cache hits: 1, misses: 1 (50.00%)
2017-Dec-26 23:16:59.017622 INFO    Closing DB.
2017-Dec-26 23:16:59.020819 ERROR   Exception: TcpListener::TcpListener, bind failed, result=49, Can't assign requested address

Does 50% mean only one of the nodes is working?
Also how can I know which vps isn't malfunctioning if that's the case?
What could be the cause of his error ?

**Comment by lanaiscool** [comment]
Are you using 2 same servers ? 
It looks like either it cant use the port you can check by checking the firewall on your vps hosting panel or check of the port is already in use another problem could be that it cant start a server on 127.0.0.1 please check your /etc/hosts

**Comment by zhmrab** [comment]
Thanks for answering, 
I am indeed using 2 identical ubuntu vps with Forknote, I actually started over from scratch and that error message is now gone but whenever I start the miner now nothing happens, 
I've been advised that it may take several hours for the miner to start but it's been 3 days and still nothing! 

Apart from that, everything else seems to be running smoothly but bc_height is still stuck at 0 :/

Have you encountered this issue before?

**Comment by lanaiscool** [comment]
yes i have are you mining on your vps ? please dont do that until block height is 3 the first 3 blocks take some time the best thing you can do is build your coin on your desktop/laptop than mine it it will take around 1-6 hours and the first 3 blocks will be mined after that it goes faster if you do not own a desktop/laptop just let it mine

**Comment by zhmrab** [comment]
Hey no I am not mining on the vps just on my computer, on the vps there's only Forknote deamon setup with the upstart conf,

After doing a little bit of digging I found out that to mine with Forknote you need to specify the deamon rpc port : 

./miner --address X --daemon-rpc-port Y

It seems to be working now thank you for taking the time to respond,
I will start working now on a GUI wallet and easy miner for windows & mac :)

**Comment by bernatixer** [comment]
I'm facing the same problem, I've started mining with the command "./miner --address X --daemon-rpc-port Y", how many time it lasted to you to find the first block?

**Comment by lanaiscool** [comment]
well i started mining after the first 5 blocks however didnt see any incoming transaction 2 days later i checked my balance and o had 35000 coins in my wallet and there were 245 blocks mined
